library/java/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -301,6 +304,12 @@ String resolveTemplate(final String configTemplate, final String platformFilterT
       configBuilder.append("] \n");
     }
 
+    // Add new anchors to override the default anchors in config header.
+    configBuilder.append("- &validation_context").append(certValidationTemplate).append("\n");
```

Comment:
Can we combine these 2 appends into one? That would be easier to read I think (may be my personal bias). 

optional comment - I do not have a strong opinion

Reply:
As discussed offline, it is okey to always use validation_context_config_trust_chain for all protocols.

Reply:
Things might change if we stop creating a new thread for each validation, but coalesce validations to a few dedicated threads.